ci/travis/37-latest-conda-forge.yaml

```diff
@@ -21,3 +21,6 @@ dependencies:
   - SQLalchemy
   - psycopg2
   - libspatialite
+  - geoalchemy2
+  # TEMP: workaround until conda-forge fixes the issues with gdal
+  - zstd
```

Comment:
This was removed recently, can you delete it here as well?

Reply:
@martinfleis Done!